### DISCUSSION TITLE
how do I run this in docker?

### DISCUSSION THREAD

**Discussion by KlavsKlavsen** [inducer/relate#895] [Tech support]
I have tried running: docker-compose up --build - under .devcontainer folder - and it spins up a pgsql container and an app container - but the app container does not listen on any port?
I would like to test this out (and later run it) - in docker - but seem to be missing some detail ?

The docs mention inducer/relate-runpy-amd64 - but that image is 3 years old - and I can't seem to run it either.
if I could just add listen port to the docker-compose config and make it run - instead of the default "sleep infinity" ?

I tried (from inside the devcontainer_app to run db initialization):
```
root@d6836302afc3:/workspace# python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 51, in <module>
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'
```

**Comment by inducer** [comment]
I don't know much about this `.devcontainer` thing, I don't use it. It was added by @joshuata at one point. Given that I'm not maintaining it, I'll remove it to avoid having it cause confusion.

#898
